languages/cpp/exercises/concept/numbers/.meta/example.cpp

```diff
@@ -0,0 +1,33 @@
+#include "numbers.h"
+
+namespace assembly_line {
+
+constexpr std::int32_t production_rate_default_speed = 221;
```

Comment:
Are students familiar with `constexpr` at this point? As per the docs, the example implementation should only use language features introduced by the exercise or its prerequisites (and their prerequisites, and so on).